Octokit.Reactive/Clients/ObservableMiscellaneousClient.cs

```diff
@@ -9,6 +9,7 @@ public class ObservableMiscellaneousClient : IObservableMiscellaneousClient
     {
         readonly IMiscellaneousClient _client;
 
+        [Obsolete("Please use another constructor")]
```

Comment:
Let's make this really specific - what about something like: `[Obsolete("Please use the IGitHubClient overload ctor")]`